scripts/release/update-spm-manifest.sh

```diff
@@ -4,52 +4,34 @@ set -euo pipefail
 
 #
 # Usage:
-#   ./scripts/release/update-spm-manifest.sh <maps version number> <common version number> <core version number> <maps xcframework checksum>
+#   ./scripts/release/update-spm-manifest.sh <maps version number> <maps xcframework checksum>
 #
 
 MAPS_VERSION=${1}
-COMMON_VERSION=${2//v/}
-CORE_VERSION=${3//v/}
-CHECKSUM=${4}
-GITHUB_TOKEN=$(./scripts/get_token.js)
+CHECKSUM=${2}
+COMMON_VERSION=`awk -F'MapboxCommon-ios.json" ==' '{ print $2 }' Cartfile`
+CORE_VERSION=`awk -F'MapboxCoreMaps.json" ==' '{ print $2 }' Cartfile`
```

Comment:
We should use `$(...)` instead of backticks: http://mywiki.wooledge.org/BashFAQ/082